updates/27/updates.sql

```diff
@@ -0,0 +1,18 @@
+UPDATE common_asset_details SET cryptocompare="USDT" WHERE identifier="eip155:534352/erc20:0xf55BEC9cafDbE8730f096Aa55dad6D22d44099Df";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:534352/erc20:0xf55BEC9cafDbE8730f096Aa55dad6D22d44099Df", "Tether USD", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:534352/erc20:0xf55BEC9cafDbE8730f096Aa55dad6D22d44099Df", "A", 534352, "0xf55BEC9cafDbE8730f096Aa55dad6D22d44099Df", 6, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:534352/erc20:0xf55BEC9cafDbE8730f096Aa55dad6D22d44099Df", "USDT", "tether", "USDT", NULL, 1696952284, NULL);
+UPDATE common_asset_details SET cryptocompare="DAI" WHERE identifier="eip155:534352/erc20:0xcA77eB3fEFe3725Dc33bccB54eDEFc3D9f764f97";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:534352/erc20:0xcA77eB3fEFe3725Dc33bccB54eDEFc3D9f764f97", "Dai Stablecoin", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:534352/erc20:0xcA77eB3fEFe3725Dc33bccB54eDEFc3D9f764f97", "A", 534352, "0xcA77eB3fEFe3725Dc33bccB54eDEFc3D9f764f97", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:534352/erc20:0xcA77eB3fEFe3725Dc33bccB54eDEFc3D9f764f97", "DAI", "dai", "DAI", NULL, 1696919918, NULL);
+UPDATE common_asset_details SET cryptocompare="DAI" WHERE identifier="eip155:250/erc20:0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:250/erc20:0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E", "Dai Stablecoin", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:250/erc20:0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E", "A", 250, "0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:250/erc20:0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E", "DAI", "dai", "DAI", NULL, 1613748662, NULL);
+UPDATE common_asset_details SET cryptocompare="USDT" WHERE identifier="eip155:250/erc20:0x049d68029688eAbF473097a2fC38ef61633A3C7A";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:250/erc20:0x049d68029688eAbF473097a2fC38ef61633A3C7A", "Frapped USDT", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:250/erc20:0x049d68029688eAbF473097a2fC38ef61633A3C7A", "A", 250, "0x049d68029688eAbF473097a2fC38ef61633A3C7A", 6, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:250/erc20:0x049d68029688eAbF473097a2fC38ef61633A3C7A", "fUSDT", "tether", "USDT", NULL, 1615367956, NULL);
```

Comment:
have you checked if it has its own coingecko/cc? since multichain got hacked I'm not sure if it is 100% backed